Driverless AI/machine-learning-experiment-scoring-and-analysis-financial-focus/machine-learning-experiment-scoring-and-analysis-financial-focus.md

```diff
@@ -1220,25 +950,25 @@ Remember that for the **ROC** curve:
 
 **Note:** If you are not sure what AUC, MCC, F1, and Accuracy are or how they are calculated review the concepts section of this tutorial.
 
-### New Model with Same Parameters
+### New Experiment with Same Settings
 
-In case you were curious and wanted to know if you could improve the accuracy of the model, you could try changing the scorer from Logloss to Accuracy. A question to keep in mind after making this change, **Does changing the scorer from Logloss to Accuracy improve the model's accuracy?**  
+In case you were curious and wanted to know if you could improve the accuracy of the model, you could try changing the scorer from Logloss to Accuracy. A question to keep in mind after making this change, **Does changing the scorer from Logloss to Accuracy improve the model's accuracy?**
 
-1\. To do this, click on the **Experiments**  page.
+1\. To do this, click on the **Experiments** page.
 
-2\. Click on the experiment you did for task 1 and select **New Model With Same Params**
+2\. Click on the experiment you did for task 1 and select **New Experiment with Same Settings**
 
-![new-model-w-same-params](assets/new-model-w-same-params.jpg)
+![new-model-w-same-params](assets/new-model-w-same-params.png)
 
-An image similar to the one below will appear. Note that this page has the same settings as the setting in Task 1. The only difference is that on the **Scorer** section **Logloss** was updated to **Accuracy**. Everything else should remain the same.
+An image similar to the one below will appear. Note that this page has the same settings as the setting in Task 1. The only difference is that on the **Scorer** section, we updated **Logloss** to **Accuracy**. Everything else should remain the same.
 
 3\. If you haven’t done so, select **Accuracy** on the scorer section then select **Launch Experiment**
 
-![new-model-accuracy](assets/new-model-accuracy.jpg)
+![new-model-accuracy](assets/new-model-accuracy.png)
 
-Similarly to the experiment in Task 1, wait for the experiment to run. After the experiment is done running, a similar page will appear. Note that on the summary located on the bottom right-side both the validation and test scores are no longer being scored by **Logloss** instead by **Accuracy**. 
+Similarly to the experiment in Task 1, wait for the experiment to run. After the experiment is done running, a similar page will appear. Note that on the summary located on the bottom right-side, both the validation and test scores are no longer being scored by **Logloss** instead by **Accuracy**. 
 
-![new-experiment-accuracy-summary](assets/new-experiment-accuracy-summary.jpg)
+![new-experiment-accuracy-summary](assets/new-experiment-accuracy-summary.png)
 
 We are going to use this new experiment to run a new diagnostics test. You will need the name of the new experiment. In this case, the experiment name is **1.Freddie Mac Classification Tutorial**. 
```

Comment:
close the space between the word diagnostics and :, located above: (6. In the **Create new model diagnostics :**)